framework/src/loops/ComputeMarkerThread.C

```diff
@@ -81,7 +81,10 @@ ComputeMarkerThread::onElement(const Elem * elem)
 }
 
 void
-ComputeMarkerThread::onBoundary(const Elem * /*elem*/, unsigned int /*side*/, BoundaryID /*bnd_id*/)
+ComputeMarkerThread::onBoundary(const Elem * /*elem*/,
+                                unsigned int /*side*/,
+                                BoundaryID /*bnd_id*/,
+                                const Elem * /*=nullptr*/)
```

Comment:
Per before let's make sure we also have a meaningful variable name in the comment in case an implement wants to come along and make use of it (although yes they should inspect the header :-))